src/services/AffectService.ts

```diff
@@ -136,19 +141,25 @@ export async function putAffectCvssScore(
 // }
 export async function postAffectCvssScore(affectId: string, cvssScoreObject: ZodAffectCVSSType) {
   const postObject: Record<string, any> = Object.assign({}, cvssScoreObject);
+  const isV2 = osimRuntime.value?.flags?.affectsV2;
+  const apiVersion = isV2 ? 'v2beta' : 'v1';
+  const path = isV2 ? 'cvss-scores' : 'cvss_scores';
   return osidbFetch({
     method: 'post',
-    url: `/osidb/api/v1/affects/${affectId}/cvss_scores`,
+    url: `/osidb/api/${apiVersion}/affects/${affectId}/${path}`,
     data: postObject,
   })
     .then(response => response.data)
     .catch(createCatchHandler('Problem updating affect CVSS scores:'));
 }
 
 export async function deleteAffectCvssScore(affectId: string, cvssScoresId: string) {
+  const isV2 = osimRuntime.value?.flags?.affectsV2;
+  const apiVersion = isV2 ? 'v2beta' : 'v1';
+  const path = isV2 ? 'cvss-scores' : 'cvss_scores';
```

Comment:
wonder if it makes sense to have isV2 or isOsidbApiV2 live in runtime as a readonly

Reply:
my thought was just to move it up so as to only have to change it on one place but probably not worth the effort since it's so little to delete

Reply:
Ooh you mean outside of the function so it can be reused, yeah 😆  that is a better approach, I though you meant exporting it directly in the `osimRuntime` the `v1`/`v2`.
Do you want me to do the change or is ok?